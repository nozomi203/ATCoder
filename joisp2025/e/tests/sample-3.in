9 4
4 3 5 3 8 1 5 8 1 7 6 2 4 9 6 9 2 7
12 9 4 8 7 1 20 5 8 7 4 13 5 9 10 3 7 8
6
39
81
73
79
64
52
