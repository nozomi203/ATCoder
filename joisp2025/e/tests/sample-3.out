1
18
3
10
1
1
